tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -578,21 +574,19 @@ def test_it_includes_the_group(self, config, context):
 
         assert groups == [context.h_group.groupid.return_value]
 
-    @pytest.mark.usefixtures("canvas_groups_on")
```

Comment:
I think this is testing https://github.com/hypothesis/lms/blob/03209a88be94ca077365dd3ba048fb1665c612e5/lms/resources/lti_launch.py#L223 instead.